src/utils/openInDefaultApp.ts

```
@@ -34,30 +41,129 @@ export function buildOpenCommand(
+    return;
+  }
+
+  await new Promise<void>((resolve) => {
+    let settled = false;
+    const finish = () => {
+      if (settled) return;
+      settled = true;
+      resolve();
+    };
+
+    let child;
```

Biome (2.4.16)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)